logstash-core/src/test/java/org/logstash/config/ir/CompiledPipelineTest.java

```diff
@@ -47,16 +51,15 @@
 import org.logstash.RubyUtil;
 import org.logstash.common.IncompleteSourceWithMetadataException;
 import org.logstash.common.SourceWithMetadata;
-import org.logstash.config.ir.compiler.AbstractFilterDelegatorExt;
-import org.logstash.config.ir.compiler.AbstractOutputDelegatorExt;
-import org.logstash.config.ir.compiler.FilterDelegatorExt;
-import org.logstash.config.ir.compiler.PluginFactory;
+import org.logstash.config.ir.compiler.*;
```

Comment:
let's avoid these refactors, I think we benefit from having an explicit list, makes it easier to understand the dependencies. Intellij can be [told to stop doing this](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports2).